tests/integration/test_nft_creation.py

```diff
@@ -19,27 +20,23 @@
 from ocean_lib.web3_internal.wallet import Wallet
 from web3 import Web3
 
-_NETWORK = "development"
+_NETWORK = "ganache"
 
 
 def get_nft_factory_address(config):
     """Helper function to retrieve a known ERC721 factory address."""
 
-    # FIXME: fix get_contract_addresses bug to use here
-    with open(config.address_file) as f:
-        network_addresses = json.load(f)
-
-    return network_addresses[_NETWORK][ERC721FactoryContract.CONTRACT_NAME]
+    return get_contracts_addresses(_NETWORK, config.address_file)[
+        ERC721FactoryContract.CONTRACT_NAME
+    ]
 
 
 def get_nft_template_address(config):
     """Helper function to retrieve a known ERC721 template address."""
 
-    # FIXME: fix get_contract_addresses bug to use here
-    with open(config.address_file) as f:
-        network_addresses = json.load(f)
-
-    return network_addresses[_NETWORK][ERC721Token.CONTRACT_NAME]["1"]
+    return get_contracts_addresses(_NETWORK, config.address_file)[
+        ERC721Token.CONTRACT_NAME
+    ]["1"]
```

Comment:
Just as a note here, I will add those address retrieval functions into an utils folder that exists in the `main` branch of ocean.py.